Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -928,38 +965,42 @@ lemma eLpNormEssSup_eq_iSup (hμ : ∀ a, μ {a} ≠ 0) (f : α → ε) : eLpNor
 @[simp] lemma eLpNormEssSup_count [MeasurableSingletonClass α] (f : α → ε) :
     eLpNormEssSup f .count = ⨆ a, ‖f a‖ₑ := essSup_count _
 
-theorem MemLp.left_of_add_measure [TopologicalSpace ε] {f : α → ε} (h : MemLp f p (μ + ν)) :
+theorem MemLp.left_of_add_measure {f : α → ε} (h : MemLp f p (μ + ν)) :
```

Comment:
`ContinuousENorm` was no necessary before, but now is?

Reply:
This PR actually strengthens some hypotheses from `[TopologicalSpace F] [ENorm F]` to `[TopologicalSpace F] [ContinuousENorm F]` (where `F` should be `\epsilon`). The question is: does that matter in practice?
(With the definition `Integral`, my initial generalisation also did not assume the enorm was continuous, which caused failures down the line - so it's certainly simpler to just assume the enorm is always continuous. Do you gain anything by removing that assumption? I'm willing to work to make mathlib more general, but there should be a point to it.)

Reply:
No, I don't think it matters really. I was just surprised that this wasn't flagged as an unused argument.